package.json

```diff
@@ -62,6 +62,7 @@
     "localforage": "^1.10.0",
     "miragejs": "^0.1.47",
     "mocha": "^5.2.0",
+    "moment": "^2.24.0",
```

Comment:
FOLIO is trying to drop moment and recommends dayjs or other similar libraries instead